src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -67,7 +71,7 @@ case class Profile (
 object Profile {
 
   // increment this number every time you make a change to the user-provided profile fields
-  val currentVersion:Int = 2
+  val currentVersion:Int = 3
```

Comment:
(But I don't prefer it enough to insist on it, which is why I didn't insist on it.)